account_bank_statement_import_coda/__openerp__.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#     This file is part of account_bank_statement_import_coda,
+#     an Odoo module.
+#
+#     Copyright (c) 2015 ACSONE SA/NV (<http://acsone.eu>)
+#
+#     account_bank_statement_import_coda is free software:
+#     you can redistribute it and/or modify it under the terms of the GNU
+#     Affero General Public License as published by the Free Software
+#     Foundation,either version 3 of the License, or (at your option) any
+#     later version.
+#
+#     account_bank_statement_import_coda is distributed
+#     in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+#     even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+#     PURPOSE.  See the GNU Affero General Public License for more details.
+#
+#     You should have received a copy of the GNU Affero General Public License
+#     along with account_bank_statement_import_coda.
+#     If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'Import CODA Bank Statement',
+    'author': "ACSONE SA/NV,Odoo Community Association (OCA)",
+    'website': "http://www.acsone.eu",
+    'category': 'Accounting & Finance',
+    'version': '1.0',
+    'license': 'AGPL-3',
+    'depends': [
+        'account_bank_statement_import',
+        'l10n_be_coda'
```

Comment:
Some don't use l10n_be and l10n_be_coa_multilang instead.

By same model, you mean the "CODA Notes" field? I always thought it was weird to have a specific field for that. I'd say it could be a generic field on statement or statement line that holds additional information and could be used by any parser. So not CODA specific.